packages/backend/jest.config.js

```diff
@@ -7,7 +7,7 @@ const packageName = require('./package.json').name
 module.exports = {
   ...baseConfig,
   clearMocks: true,
-  testTimeout: 15000,
+  testTimeout: 30000,
```

Comment:
I'm guessing once we have psql accounting by default, we can revert to 15000 and only set 30000 for the tigerbeetle tests?

Reply:
I think so. I tried reverting already once I figured out where the problem was but 15 seconds wasn't enough.